<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KlasaGlowna</class>
 <widget class="QMainWindow" name="KlasaGlowna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Telemetrii</string>
  </property>
  <property name="windowIcon">
   <iconset resource="zasoby.qrc">
    <normaloff>:/ikony/zasoby/icon.png</normaloff>:/ikony/zasoby/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="Okno3D" name="openGLWidget_okno3d"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="actionArtificial_horizon"/>
   <addaction name="actionControl"/>
   <addaction name="actionTerminal"/>
   <addaction name="actionOptions"/>
   <addaction name="action3D_View"/>
  </widget>
  <widget class="QDockWidget" name="dock_terminal">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTextBrowser" name="terminal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_artificial_horizon">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="sztuczny_horyzont" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_control">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="verticalSlider_x">
       <property name="maximum">
        <number>5760</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>2880</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="verticalSlider_y">
       <property name="maximum">
        <number>5760</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>2880</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="verticalSlider_z">
       <property name="maximum">
        <number>5760</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>2880</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuWidget">
     <property name="title">
      <string>Widget</string>
     </property>
     <addaction name="actionArtificial_horizon"/>
     <addaction name="actionTerminal"/>
     <addaction name="actionControl"/>
     <addaction name="actionOptions"/>
     <addaction name="action3D_View"/>
    </widget>
    <addaction name="menuWidget"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="dock_options">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_load_model">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Wczytaj Model 3D</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="zasoby.qrc">
     <normaloff>:/ikony/zasoby/close.png</normaloff>:/ikony/zasoby/close.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Close program</string>
   </property>
  </action>
  <action name="actionArtificial_horizon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="zasoby.qrc">
     <normaloff>:/ikony/zasoby/artificial horizon.png</normaloff>:/ikony/zasoby/artificial horizon.png</iconset>
   </property>
   <property name="text">
    <string>artificial horizon</string>
   </property>
  </action>
  <action name="actionControl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="zasoby.qrc">
     <normaloff>:/ikony/zasoby/control.png</normaloff>:/ikony/zasoby/control.png</iconset>
   </property>
   <property name="text">
    <string>control</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="zasoby.qrc">
     <normaloff>:/ikony/zasoby/terminal.png</normaloff>:/ikony/zasoby/terminal.png</iconset>
   </property>
   <property name="text">
    <string>terminal</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="zasoby.qrc">
     <normaloff>:/ikony/zasoby/options.png</normaloff>:/ikony/zasoby/options.png</iconset>
   </property>
   <property name="text">
    <string>options</string>
   </property>
  </action>
  <action name="action3D_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="zasoby.qrc">
     <normaloff>:/ikony/zasoby/3D view.png</normaloff>:/ikony/zasoby/3D view.png</iconset>
   </property>
   <property name="text">
    <string>3D View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Okno3D</class>
   <extends>QOpenGLWidget</extends>
   <header>okno3d.h</header>
  </customwidget>
  <customwidget>
   <class>sztuczny_horyzont</class>
   <extends>QGraphicsView</extends>
   <header>sztuczny_horyzont.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="zasoby.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>KlasaGlowna</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionControl</sender>
   <signal>toggled(bool)</signal>
   <receiver>dock_control</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTerminal</sender>
   <signal>toggled(bool)</signal>
   <receiver>dock_terminal</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionArtificial_horizon</sender>
   <signal>toggled(bool)</signal>
   <receiver>dock_artificial_horizon</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dock_artificial_horizon</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionArtificial_horizon</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dock_control</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionControl</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dock_terminal</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionTerminal</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dock_options</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionOptions</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOptions</sender>
   <signal>toggled(bool)</signal>
   <receiver>dock_options</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action3D_View</sender>
   <signal>toggled(bool)</signal>
   <receiver>centralWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>centralWidget</sender>
   <signal>destroyed()</signal>
   <receiver>action3D_View</receiver>
   <slot>hover()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
